docs/process_manual/2020/advice.md

```diff
@@ -0,0 +1,38 @@
+# Advice for Future Process Groups
+
+This section will describe our views and experiences, and what we found out that worked. 
+Previous advice can be found [here](../../getting_started/advice_for_future_giraf/process_advice.md).
+
+## The Process in General
+
+* The project is everyone's, the process and PO group does not have more authority than everyone else, just more responsibility.
+* Take your time to understand the process and project, read the wiki, it is well spent.
+* Use the process exactly as described on this wiki and then modify it, to avoid confusion.
+* Use Gitflow, as described in the wiki.
+* Be aware of the process changeing and make an active decision about what to do.
+* Use the tools provided on this wiki and improve them for yourselves and the next years.
+* Use some time at the end of your project making it ready to hand off to the next year.
+    * An important way to do this, is to keep Github clean of unused branches throughout the project.
+
+## Communication
+
+* Use Discord or Slack as the communication medium. Use it to broadcast event start times and deadlines.
+* Events should be held at a physical location with all developers present.
```

Comment:
Physical and/or remote location maybe? Since we have used Discord due to Covid-19.